api/envoy/config/route/v3/route_components.proto

```diff
@@ -745,6 +745,9 @@ message RouteAction {
     message ConnectConfig {
       // If present, the proxy protocol header will be prepended to the CONNECT payload sent upstream.
       core.v3.ProxyProtocolConfig proxy_protocol_config = 1;
+
+      // If set, the TCP proxy will expect POST payload as raw TCP.
```

Comment:
comment is wrong. This is about the route.

Shall we name this `allow_post` and if it is true allow both POST and CONNECT be decapped?